v21.2/serverless-function-best-practices.md

```diff
@@ -0,0 +1,115 @@
+---
+title: Serverless Function Best Practices
+summary: Best practices for optimizing the performance of serverless functions (e.g., AWS Lambda functions or Google Cloud Functions) that connect to CockroachDB.
+toc: true
+referral_id: docs_serverless_functions
+---
```

Comment:
Don't forget to add `docs_area`: https://github.com/cockroachdb/docs/blob/master/CONTRIBUTING.md#pages

Reply:
Done.